src/components/Logo/logo.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
SAME!!!!! ADD COMMENTS

Reply:
Name of component should also be good. LOGO is not a proper name. Please discuss with team mates before creating the components.

Reply:
I think logo is the right name because we are rending only our brand logo with this component i dont thing there is any requirement to change it